libnavigator/src/main/java/com/mapbox/navigation/navigator/internal/NavigatorMapper.kt

```diff
@@ -328,6 +341,63 @@ internal class NavigatorMapper {
                     .alertGeometry(alert.getAlertGeometry())
                     .build()
             }
+            RouteAlertType.KINCIDENT -> {
```

Comment:
Could you add the mapper test for this type?

Reply:
in progress 🚀

Reply:
done